lib/MetricsConsumer.js

```diff
@@ -74,12 +74,13 @@ class MetricsConsumer {
                 type: 'processed'
             }
         */
+        const site = data.site;
         if (data.type === 'processed') {
-            this._sendRequest(redisKeys.opsDone, data.ops);
-            this._sendRequest(redisKeys.bytesDone, data.bytes);
+            this._sendRequest(`${redisKeys.opsDone}:${site}`, data.ops);
```

Comment:
Following the API route organization, I would rather put the `site` argument in front in the key.